modules/update-upgrading-cli.adoc

```diff
@@ -59,7 +55,9 @@ For details and information on how to perform an `EUS-to-EUS` channel update, pl
 _Preparing to perform an EUS-to-EUS upgrade_ page, listed in the Additional resources section.
 ====
 
-. Based on your organization requirements, set the appropriate update channel. For example, you can set your channel to `stable-4.12`, `fast-4.12`, or `eus-4.12`. For more information about channels, refer to _Understanding update channels and releases_ listed in the Additional resources section.
+. Based on your organization requirements, set the appropriate update channel. For example, you can set your channel to `stable-4.13` or `fast-4.13`. For more information about channels, refer to _Understanding update channels and releases_ listed in the Additional resources section.
```

Comment:
```suggestion
. Based on your organization's requirements, set the appropriate update channel. For example, you can set your channel to `stable-4.13` or `fast-4.13`. For more information about channels, refer to _Understanding update channels and releases_ listed in the "Additional resources" section.
```
- Per [ISG](https://www.ibm.com/docs/en/ibm-style?topic=references-citations#general-guidelines), "If you refer to a smaller piece of work within a larger container such as a chapter within a book or an article in a magazine, use italic font for the larger container and double quotation marks for the smaller piece."

Reply:
Resolved :)